File: app/models/cliente.py
# app/models/cliente.py
class Cliente:

    def __init__(self, id, nome, email):

        self.id = id
        self.nome = nome
        self.email = email

    def __str__(self):

        return (
            f"Cliente(id={self.id}, "
            f"nome='{self.nome}', "
            f"email='{self.email}')"
        )

    def to_dict(self):

        return {
            "id": self.id,
            "nome": self.nome,
            "email": self.email
        }

File: Cliente/app/network/java/Cliente.java
// Cliente/app/network/java/Cliente.java

package app.network.java;

public class Cliente {
    public int id;
    public String nome;
    public String email;

    public Cliente(int id, String nome, String email) {
        this.id = id;
        this.nome = nome;

        this.email = email;
    }

    public String toJson() {
        return String.format(
            "{\"id\":%d,\"nome\":\"%s\",\"email\":\"%s\"}",
            id, nome, email
        );
    }
}